src/main/kotlin/net/atos/zac/zoeken/model/zoekobject/ZaakZoekObject.kt

```diff
@@ -0,0 +1,212 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.zoeken.model.zoekobject
+
+import net.atos.client.zgw.zrc.model.Rol
+import net.atos.zac.zoeken.model.ZaakIndicatie
+import nl.lifely.zac.util.NoArgConstructor
+import org.apache.solr.client.solrj.beans.Field
+import java.util.Date
+import java.util.EnumSet
+import java.util.function.Supplier
+import java.util.stream.Collectors
+
+@NoArgConstructor // required for Java bean inspection
+data class ZaakZoekObject(
+    /**
+     * The UUID of the zaak.
+     */
+    @Field
+    var id: String,
+
+    @Field
+    private var type: String,
```

Comment:
Why bother with this field?

Reply:
Because it is added to the Solr index using the @Field annotation as far I know.

Reply:
Ah I see, no drama, I couldn't work out why it was required.